gcloud-java-core/src/main/java/com/google/gcloud/BaseServiceException.java

```diff
@@ -123,7 +138,7 @@ public BaseServiceException(GoogleJsonError error, boolean idempotent) {
     this.code = error.getCode();
```

Comment:
Seems viable. However, then we do not actually use the suggested `protected boolean isRetryable(boolean idempotent, Error error)` anywhere. Is there a reason for keeping it?